src/parsers/transformer.rs

```diff
@@ -323,27 +316,23 @@ fn fake_company_name(
     prepend_unique_if_present(new_company_name, args, unique)
```

Comment:
why do we still need this if we're no longer having the unique option

Reply:
This is used in other transformers